charts/ocis/templates/frontend/deployment.yaml

```diff
@@ -98,9 +98,13 @@ spec:
             - name: FRONTEND_FULL_TEXT_SEARCH_ENABLED
               value: {{ not (eq .Values.services.search.extractor.type "basic") | quote }}
 
+            - name: FRONTEND_SHOW_USER_EMAIL_IN_RESULTS
+              value: {{ .Values.features.sharing.users.search.showUserEmail | quote }}
+
             - name: OCIS_DISABLE_SSE
               value: {{ .Values.features.sse.disabled | quote }}
 
+
```

Comment:
Why to we have this additional whitespace here?

Reply:
fixed https://github.com/owncloud/ocis-charts/pull/526/commits/a771ac9debf6b5b8c495fb0b5bed686021206da7